### DISCUSSION TITLE
v10-(1403-7-25)

### DISCUSSION THREAD

**Discussion by GFW-knocker** [GFW-knocker/MahsaNG#92] [Announcements]
support IPv6 from MahsaServer
support hysteria2/hy2 protocol
categorize configs into subscription
Dual Async ping of google/gstatic
change widget icon to M
set google DoH as default DNS
add "clear test result" button
better handling of right-to-left device
and much UI improvements

fix warp key fetch on persian or old device
fix rare UI crash when showing progressBar

good to know:
1- hysteria2 works better on IPv6 (seems its blocked on IPv4 in some range)
2- hysteria2 plugin not work if you enable "per-app-proxy" in VPN mode. in such case you should exclude MahsaNG package to bypass that plugin. 

<hr /><em>This discussion was created from the release <a href='https://github.com/GFW-knocker/MahsaNG/releases/tag/v10-(1403-7-25)'>v10-(1403-7-25)</a>.</em>

**Comment by rasolAAaa** [comment]
عالی است

**Comment by sadeghhaghgou** [comment]
خیلی ممنون

**Comment by mahxray** [comment]
سلام لطفا میشه آخرین نسخه libv2ray.arr  ک داخل اپ‌مهسا استفاده شده برام ارسال کنید

**Reply by GFW-knocker** [reply]
https://github.com/GFW-knocker/AndroidLibXrayLite/releases/latest

**Comment by ddaniyalb** [comment]
لطفا نسخه windows هم بدید❤️عالی

**Comment by Nimavao** [comment]
لطفا نسخه دسکتاپ رو موجود کنید
